src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -168,6 +167,19 @@ public static void main(final String[] args) {
     }
   }
 
+  private static JFrame newMainFrame() {
+    final JFrame frame = new JFrame("TripleA");
+
+    frame.add(new MainPanel(setupPanelModel));
```

Comment:
Changed from

```java
frame.getContentPane().add(contents, BorderLayout.CENTER)
```

to

```java
frame.add(contents)
```

As the latter is equivalent to former since Java 6.